packages/editor/src/editor/browser/data-class/variable-tree-data.test.ts

```diff
@@ -177,3 +185,61 @@ test('return only the prefix when variablePath is empty', () => {
 test('return an empty #{ } when both prefix and variablePath are empty', () => {
   expect(formatVariableValue('')).toBe('#{}');
 });
+
+describe('collectNodesWithChildren', () => {
+  test('returns empty array for empty input', () => {
+    const result = collectNodesWithChildren([]);
+    expect(result).toEqual([]);
+  });
+
+  it('collects single-level nodes with children', () => {
```

Comment:
please do not mix `test` an `it`
even if they do the same, it is not consistent in the codebase...